Library/Homebrew/cmd/cleanup.rb

```diff
@@ -111,7 +111,11 @@ def cleanup_lockfiles
   end
 
   def rm_DS_Store
-    quiet_system "find", HOMEBREW_PREFIX.to_s, "-name", ".DS_Store", "-delete"
+    %w[Cellar Frameworks Library bin etc include lib opt sbin share var].each do |path|
+      path = HOMEBREW_PREFIX/path
+      next unless path.exist?
+      quiet_system "find", path.to_s, "-name", ".DS_Store", "-delete"
```

Comment:
> You could give all paths to find to speed up the process

Why it will speed up? It's the same amount folders which will be scanned.

> quiet_system "find", paths * " ", "-name", ".DS_Store", "-delete"

This wouldn't work. It should be:

```ruby
args = paths
args << %w[-name .DS_Store -delete]
quiet_system "find", *args
```